### PR TITLE
[WNMGDS-376] - Add accessibility page to doc site 

### PR DESCRIPTION
**Pull request by line47**
**Note:** 
- Add a page to the doc site explaining how we approach and test for accessibility.

### CODE REVIEW

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- include accessibility as part of our design process when working on new or existing components.
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: maybe this bullet and the one below can be combined? both are talking about new components

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- provide accessibility as a [checkpoint in new components](https://github.com/CMSgov/design-system/blob/master/.github/ISSUE_TEMPLATE/component-validation-checklist.md) created in the design system.
+- add accessibility as a key aspect to much of the [component documentation](https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md).
+- conduct regular accessibility reviews of new and existing features and documentation for the design system.
+
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: We should probably also add our e2e testing that uses axe for automated a11y testing. Also locally we use that browser bookmarklet for a11y as well

Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: Hey @bernardwang which browser bookmarklet are you referring to?

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 1 above)
Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: Maybe updating "include accessibility as part of our design process when working on new or existing components." to something like "include accessibility as part of our design process." would still give enough details and also allow us to keep both bullet points?

Comment (bernardwang) [reply packages/docs/src/pages/guidelines/accessibility.md]: Maybe... I feel like this bullet doesn't really say anything specific. And  honestly, accessibility should be included in design/product/engineering processes as well. The way USWDS handles this is by linking to [Accessibility for teams](https://accessibility.digital.gov/) which has some details and guidance for different roles.

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 2 above)
Comment (bernardwang) [reply packages/docs/src/pages/guidelines/accessibility.md]: This one: https://khan.github.io/tota11y/
Screen Shot 2020-03-09 at 10 23 12 AM

Although in retrospect, I don't think we should mention it. It was legacy tool (we should consider removing it even), and `axe-webdriver` in e2e testing and `axe` browser extension for web testing are much more comprehensive.
Maybe we could even link to Ad Hoc's a11y tool guidance for more info here? https://adhocteam.us/topics/accessibility/index.html

**Comment by bernardwang** [comment]
Sorry for all the content feedback! I should have posted this on the google doc first!

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- include accessibility as part of our design process when working on new or existing components.
+- provide accessibility as a [checkpoint in new components](https://github.com/CMSgov/design-system/blob/master/.github/ISSUE_TEMPLATE/component-validation-checklist.md) created in the design system.
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: Ahh, I'm conflicted on this. Accessibility is included in that issue template, but we never incorporate that into our process. Issue templates are only used by external people on github, and we've never seen anyone file an issue with that template, or with code introducing a new component. The issue template almost feels like false advertising, or a process that is technically true, but in practice meaningless.

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 1 above)
Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: I like the idea of linking out to the accessibility for teams site - I'll include that!
Maybe we can remove the 2nd bullet point altogether then?

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 3 above)
Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: I think @davidakennedy had some plans for this accessibility checklist unless I'm mistaken?

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 2 above)
Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: I'm adding in a line about using "automated accessibility testing using [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs)" but don't think we should link out to the Ad Hoc a11y page

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: oops I thought I posted this already, but i guess not

The guiding principles page only mentions accessibility once, and it feels like a really outdated guide. It has very specific info on class naming and principles behind css utilities, but nothing on a more general product principles like accessibility. I wonder if we should remove this bullet for now and work on writing a new guiding principles to include in our doc site or in the GH wiki

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Accessibility
+---
+
+CMS strives to make its Design System work for as many people as possible, including people with disabilities.
+
+## What we do to help meet and exceed standards
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: Could we capitalize the bullet points?

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- build accessibility into our [guiding principles](https://github.com/CMSgov/design-system/blob/master/guides/GUIDING-PRINCIPLES.md).
+- provide accessibility as a [checkpoint in new components](https://github.com/CMSgov/design-system/blob/master/.github/ISSUE_TEMPLATE/component-validation-checklist.md) created in the design system.
+- add accessibility as a key aspect to much of the [component documentation](https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md).
+- conduct regular accessibility reviews of new and existing features and documentation for the design system.
+- Embed [accessibility and inclusive design practices](https://accessibility.digital.gov/) into our workflow.
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: Could we move this to be first or second bullet point?

**Comment by bernardwang** [comment]
@line47 sorry again for all the changes, but maybe it would be better for me to give my feedback on the google doc?

File: packages/docs/src/pages/guidelines/accessibility.md (hunk 4 above)
Comment (line47) [reply packages/docs/src/pages/guidelines/accessibility.md]: I think that works

**Comment by bernardwang** [comment]
@line47 I think the content in [the doc](https://docs.google.com/document/d/1mTizaepejESHUiyOStSqX5W1f5kHviYhp3ruhqL93xQ/edit#) is more finalized, would you be able to update with this PR?

**Comment by line47** [comment]
I think this is ready for another pass @bernardwang and @davidakennedy 👍

File: packages/docs/src/pages/guidelines/accessibility.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+More information on Section 508 can be found at:
+
+- [United States Access Board](http://www.access-board.gov/508.htm)
+- [Section508.gov](http://www.section508.gov)
+- [HHS Digital Communications Division](http://www.hhs.gov/web/508)
+
+## Current browser/technology support
+
+The current browser support for the design system [can be viewed here](https://design.cms.gov/startup/browser-support/). We test with the following assistive technology pairings:
```
Comment (bernardwang) [inline packages/docs/src/pages/guidelines/accessibility.md]: This should be `{{root}}` instead of `https://design.cms.gov`

**Comment by line47** [comment]
Updated to use `{{root}}`

**Pull request merged**